pgml-apps/discord-bot/ingest.ipynb

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I think this should be refactored as something like `reindex.py`. It's reasonable to have a notebook for illustration/example purposes to, but it'd be good to have a build/run script.